README.md

```diff
@@ -266,7 +266,7 @@ return (
 
 <br>
 
-Let's build out the component hierarchy. We are going to implement basic CRUD functionality; **create**, **read***, **update**, **delete**. Our `Main` component could render a `Header` and a `Body`. In the `Body`, we need to be able to view all skills, create a new skill, edit a skill and delete a skill. So, `Body` could render `<NewSkill />` and `<AllSkills />`. `NewSkill` is a form to create new skills and `AllSkills` renders a collection of individual `Skill` components - each `Skill` component has it's own delete and edit button.
+Let's build out the component hierarchy. We are going to implement basic CRUD functionality; **c**reate, **r**ead, **u**pdate, **d**elete. Our `Main` component could render a `Header` and a `Body`. In the `Body`, we need to be able to view all skills, create a new skill, edit a skill and delete a skill. So, `Body` could render `<NewSkill />` and `<AllSkills />`. `NewSkill` is a form to create new skills and `AllSkills` renders a collection of individual `Skill` components - each `Skill` component has it's own delete and edit button.
```

Comment:
Yes, there is a small change. As it stands the markdown formatting is incorrect because there is one asterik out of place. I fixed that and made it so only the first letters are bold so it highlights that CRUD is an acronym.

Before:
<img width="816" alt="screen shot 2017-02-06 at 1 59 30 pm" src="https://cloud.githubusercontent.com/assets/14339707/22666317/4e94c81e-ec75-11e6-8e1c-86d71ddcc4eb.png">
After:
<img width="809" alt="screen shot 2017-02-06 at 2 03 05 pm" src="https://cloud.githubusercontent.com/assets/14339707/22666326/546aa9f2-ec75-11e6-8ce2-3f44b2fd7b63.png">

If you don't like the change I won't be offended. Thank you for the great tutorial. =)